software_reseller/data/product_product.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+    <data noupdate="1">
+        <record id="software_reseller.product_product_1_8d131916" model="product.product" >
```

Comment:
Why do you need to create this? Can't you just remove the '{'create_product_product': False}' in the product_template data and they will be created?